main.go

```diff
@@ -33,19 +48,68 @@ func main() {
 		_ = resp.Body.Close()
 	}()
 
+	// Check that the domain specified by the user exists in GoDaddy
+	domainExists := 0
 	for _, zone := range zones {
 		if zone.Status == "ACTIVE" {
-			log.Println(zone.Domain)
+			if *userDomainName == zone.Domain {
+				domainExists = 1
+				log.Println("Domain", zone.Domain, "matches user domain", *userDomainName)
+			}
 		}
 	}
 
+	if domainExists != 1 {
+		log.Println("Domain", *userDomainName, "does not match any domains returned from the API")
+	}
+
+	var recordData map[string]string
+	recordData = make(map[string]string)
+
 	records, _, err := apiClient.V1Api.RecordGet(ctx, "lwts.org", "", "", nil)
 	for _, record := range records {
-		log.Println(record.Name, "=>", record.Data)
+		if record.Name == "@" {
+			if record.Type_ == "A" {
+				log.Println("Changing", record.Type_, record.Data, "to", *userIPAddress)
+				// Fill in the recordData map so it can be used outside this loop
+				recordData["type"] = "A"
+				recordData["name"] = "@"
+
+			}
+		}
+		//log.Println(record.Name, "=>", record.Data)
 	}
 
-	domainInfo, _, err := apiClient.V1Api.Get(ctx, "lwts.org", nil)
-	log.Println(domainInfo)
+	if *applyBool {
+		var hosts []godaddy.DnsRecordCreateTypeName
+		changeData := godaddy.DnsRecordCreateTypeName{Data: *userIPAddress, Ttl: 600}
+		changeDataArray := append(hosts, changeData)
+		result, _ := apiClient.V1Api.RecordReplaceTypeName(ctx, "lwts.org", "A", "@", changeDataArray, nil)
+		if result.StatusCode == 200 {
+			log.Println("Changed", recordData["type"], recordData["name"], "to", *userIPAddress)
+		} else {
```

Comment:
Stop using `else` so much. It's just no go-idiomatic. Find another way. Use `break`, use `continue`. Set defaults first. Whatever it takes, avoid `else`.

Reply:
I got rid of one of the else's but i'm stuck on how to eliminate the first one. How can break or continue be used instead of if/else? This isn't inside a loop